ccm_web/client/src/models/FeatureUIData.tsx

```diff
@@ -76,5 +77,25 @@ const allFeatures: FeatureUIGroup[] = [
   { id: 'Sections', title: 'Sections', ordinality: 3, features: [mergeSectionCardProps, createSectionsCardProps] }
 ]
 
+const isAuthorizedForRoles = (userRoles: RoleEnum[], requiredRoles: RoleEnum[], featureDescriptionForLogging: string): boolean => {
+  const rolesAllowingAccess = userRoles.filter(requiredRole => requiredRoles.includes(requiredRole))
+  // These log messages seem useful while debugging/testing but should be removed later
+  if (rolesAllowingAccess.length > 0) {
+    console.debug('Feature ' + featureDescriptionForLogging + ' allowed by ' + rolesAllowingAccess.join(','))
+  } else {
+    console.debug('Feature ' + featureDescriptionForLogging + ' forbid for ' + userRoles.join(','))
+  }
```

Comment:
So do you think we'll remove these before merging, or do you want to keep them between PRs?

Reply:
I think it will be useful for anyone doing testing to be able to look at the console to see why they were able or not able to see a feature.  I suppose I'd just leave it and have a cleanup issue to remove it later?

Reply:
Sure, that's fine with me.

Reply:
Created #134